Passwords/10_million_password_list_top_1000.txt

```diff
@@ -344,7 +344,6 @@ blue
 liverpool
 theman
 bandit
```

Comment:
But this fixes problem only on your side, meanwhile everyone else still need to be safe.
So I wait from this PR that all passwords will be deleted.

Reply:
Steve Gibson is on the job, https://www.grc.com/sqrl/sqrl.htm

Reply:
interesting found my name exists in username list